x-pack/plugins/stack_alerts/public/rule_types/es_query/expression/es_query_expression.tsx

```diff
@@ -17,13 +17,22 @@ import { XJson } from '@kbn/es-ui-shared-plugin/public';
 import { CodeEditor, useKibana } from '@kbn/kibana-react-plugin/public';
 import { getFields, RuleTypeParamsExpressionProps } from '@kbn/triggers-actions-ui-plugin/public';
 import { parseDuration } from '@kbn/alerting-plugin/common';
+import {
+  FieldOption,
+  buildAggregation,
+  parseAggregationResults,
+  isGroupAggregation,
+  isCountAggregation,
+  BUCKET_SELECTOR_FIELD,
+} from '@kbn/triggers-actions-ui-plugin/public/common';
+import { Comparator } from '../../../../common/comparator_types';
+import { getComparatorScript } from '../../../../common';
```

Comment:
btw, async chunks size of `stack_alerts` increased. Last time I was dealing with it was due to `@kbn-config/schema` dependency which pulls `joi` package. You can see it if you add webpack analyse label. So, as long as you have any import from `x-pack/plugins/stack_alerts/common/comparator.ts` file with `@kbn-config/schema`, you will have such a huge async chunk size.

![924A5442-7E73-4AE8-BFFE-3D20DDA4C479_4_5005_c](https://user-images.githubusercontent.com/39378793/203573040-2cac2e6e-98ca-4678-83c5-c0c689952d7e.jpeg)

Reply:
Thanks for the tip! Fixed in https://github.com/elastic/kibana/pull/144689/commits/546dd7ed5a05789bf29650ab64e9547a7c232d6f